### ISSUE TITLE
gsutil ls -L fails in 5.10

### ISSUE THREAD

**Issue by rjherman2011** [GoogleCloudPlatform/gsutil#1508]
Title says it. Failing on 5.10, but not older versions
![image](https://user-images.githubusercontent.com/4885202/166720343-e69ffb9d-892a-46ec-9a7b-b7204c84b90c.png)

**Comment by PhilBrammer** [comment]
@rjherman2011 is on an Ubuntu 18.04 LTS box using a GCP pre-built image.

Rolling back to gcloud sdk 383.0.1 works (gsutil 5.9):
```
# /snap/bin/gcloud --version
Google Cloud SDK 383.0.1
alpha 2022.04.26
beta 2022.04.26
bq 2.0.74
bundled-python3-unix 3.8.11
core 2022.04.26
gsutil 5.9
minikube 1.25.2
skaffold 1.38.0
```
```
# snap list google-cloud-sdk --all
Name              Version  Rev  Tracking         Publisher          Notes
google-cloud-sdk  383.0.1  237  latest/stable/…  google-cloud-sdk✓  classic
google-cloud-sdk  384.0.0  238  latest/stable/…  google-cloud-sdk✓  disabled,classic
```

Additionally, when on 384, while `gcloud --version` lists gsutil as 5.10, gsutil itself is noting 5.6:
```
# snap revert --revision=238 google-cloud-sdk 
google-cloud-sdk reverted to 384.0.0
# /snap/bin/gsutil --version
gsutil version: 5.6
```

**Comment by thomasmaclean** [comment]
The reason for this is that `384.0.0` shipped with `5.6`, and is incorrectly reporting `5.10` in the `gcloud version` output as well as the Cloud SDK changelog. You can confirm this by running `gsutil version`, which will output `5.6` on Google Cloud SDK 384.0.0. A hotfix (`384.0.1`) has been released that will correct the gsutil version distributed with Cloud SDK.

**Comment by PhilBrammer** [comment]
Sad face. And snap auto updates killed us here.
